### DISCUSSION TITLE
call method from wire:model

### DISCUSSION THREAD

**Discussion by frenchqwerty** [livewire/livewire#2737] [1. Help]
Hello,

Is it possible to call a method from a wire:model?

`<input type="checkbox" class="mr-1" wire:model="elementIsSelected('{!! $id !!}')"/>`

**Comment by calebporzio** [comment]
Sorry no, that doesn't really fit the paradigm of wire:model. You could listen for wire:input instead

**Reply by manishbasnyat182** [reply]
Why isn't this mentioned in the doc @calebporzio ??? I haven't switched to v3 in my old projects, and not knowing this was a hassle......
but thank you so much for this magic method call !!!
